flow-decomposition/src/test/java/com/powsybl/flow_decomposition/FlowDecompositionObserverTest.java

```diff
@@ -329,15 +329,34 @@ void testRemoveObserver() {
         assertTrue(reportRemoved.allEvents().isEmpty());
     }
 
+    @Test
+    void testObserverWithEnableLossesCompensation() {
+        String networkFileName = "19700101_0000_FO4_UX1.uct";
+        String branchId = "DB000011 DF000011 1";
+        Network network = TestUtils.importNetwork(networkFileName);
+        XnecProvider xnecProvider = XnecProviderByIds.builder()
+                .addNetworkElementsOnBasecase(Set.of(branchId))
+                .build();
+        var flowDecompositionParameters = FlowDecompositionParameters.load().setEnableLossesCompensation(true);
+        FlowDecompositionComputer flowComputer = new FlowDecompositionComputer(flowDecompositionParameters);
+        var report = new ObserverReport();
+        flowComputer.addObserver(report);
+        flowComputer.run(xnecProvider, network);
+
+        // there are no losses in acNodalInjection
+        String lossesId = LossesCompensator.getLossesId("");
```

Comment:
Could you extract a constant "LOSSES" in LossesCompensator instead of using this method please ?